### PR TITLE
dockerfile: project name as build argument

### LINKED ISSUE #4: tests: add travis test
See example https://github.com/getsentry/docker-sentry/blob/master/.travis.yml

See also https://github.com/docker-library/official-images

### PR DESCRIPTION
**Pull request by dinosk**
* Adds build argument project name, to resolve properly the
  instance path.

* Adds tests directory and extends .travis.yml to assert the installation of libraries and configuration.

Closes #4 #10 

Signed-off-by: Dinos Kousidis <dinos.kousidis@cern.ch>
Co-Authored-by: Christos Topaloudis <mr.topless@cern.ch>

### CODE REVIEW

File: python3.6/Dockerfile [hunk 1]
```diff
@@ -40,30 +43,30 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 # Create working directory
-ENV WORKING_DIR=/opt/invenio
-ENV INVENIO_INSTANCE_PATH=${WORKING_DIR}/var/instance
-RUN mkdir -p ${INVENIO_INSTANCE_PATH}
+ENV PROJECT_DIR=/opt/${PROJECT_NAME}
```
Comment (dinosk) [inline python3.6/Dockerfile]: The renaming here to `PROJECT_` is made to align with the cookiecutter project name / package name.

File: python3.6/Dockerfile [hunk 2]
```diff
@@ -40,30 +43,30 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 # Create working directory
-ENV WORKING_DIR=/opt/invenio
-ENV INVENIO_INSTANCE_PATH=${WORKING_DIR}/var/instance
```
Comment (dinosk) [inline python3.6/Dockerfile]: If an instance is created with cookiecutter-invenio-instance using a name different than `invenio` this Dockerfile won't work, added the mandatory `ARG PROJECT_NAME` to address this.

**Review by kpsherva — APPROVED** [review]

**Comment by dinosk** [comment]
The updated documentation is on HackMD for easier commenting: https://hackmd.web.cern.ch/YTC1W2T8SOm-GJnFbTTfrA?both
Pushing to the PR after the review

**Review by ppanero — APPROVED** [review]
In the images maybe putting the (docker) commands that users should follow.

**Comment by dinosk** [comment]
True @ppanero, this can be added in #14 in the text in of the figure label so that the commands can be copied over if needed.

**Pull request merged**